AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -4770,3 +4770,4 @@ jianshu.com#$#.download-app-guidance { display: none !important; }
 jianshu.com#$#body { overflow: auto !important; }
 jianshu.com##.dt-open-bg[data-scene="leftBtn"]
 jianshu.com##.note-comment-above-ad-wrap
+mondaq.com##.cd-user-modal
```

Comment:
This modal pops up because:
![settimeout](https://user-images.githubusercontent.com/18536328/132256454-7cb62373-e884-4fdf-9d59-48bf68a993ab.png)

![signup](https://user-images.githubusercontent.com/18536328/132256459-33635894-e57f-4543-9d6f-c25ca0620ebf.png)

Please remove your rule and add at the top somewhere under `nestoria.de###alertBottom` rule:
`mondaq.com#%#//scriptlet('prevent-setTimeout', 'cd-user-modal')`

If you're wondering read
about scriptlets:
https://github.com/AdguardTeam/Scriptlets/blob/master/wiki/about-scriptlets.md
our knowledgebase:
https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters

Reply:
Thanks for your explanation. I can see why now the button breaks. Do you know where I can find the nestoria.de rule? I can't seem to locate it in the `annoyances.txt` file.

Reply:
@Mennaruuk please change your rule to `mondaq.com#%#//scriptlet('prevent-setTimeout', 'cd-user-modal')`

Reply:
Sorry I'm so bad at this, I need to improve!! You have a keen eye. I think it LGTM!

Reply:
No problem =)